exposed-java-time/src/test/kotlin/org/jetbrains/exposed/JavaTimeTests.kt

```diff
@@ -472,6 +472,21 @@ open class JavaTimeBaseTest : DatabaseTestsBase() {
                         .single()[testTable.timestampWithTimeZone.date()]
                 )
 
+                val expectedTime =
+                    when (testDb) {
+                        TestDB.SQLITE -> OffsetDateTime.parse("2023-05-04T05:04:01.123+00:00")
+                        TestDB.MYSQL_V8, TestDB.SQLSERVER,
+                        TestDB.H2_V2_ORACLE, TestDB.ORACLE,
+                        TestDB.H2_V2_PSQL, TestDB.POSTGRESQL, TestDB.POSTGRESQLNG ->
+                            OffsetDateTime.parse("2023-05-04T05:04:01.123123+00:00")
```

Comment:
`TestDB.ALL_ORACLE_LIKE` and `TestDB.ALL_POSTGRES_LIKE` would simplify this branch.